src/dirwatcher.js

```diff
@@ -0,0 +1,58 @@
+import { execSync } from 'child_process';
+import { EventEmitter } from 'events'
+import path from 'path';
+import csvjson from 'csvjson';
+import Importer from './importer';
+
+const DEFAULT_DELAY = 500;
+
+export default class DirWatcher extends EventEmitter {
+  /**
+   * Declaring properties
+   */
+  constructor() {
+    super();
+    this.interval = null;
+    this.files = [];
+  }
+
+  /**
+   * Watches directory and emit 'dirwatcher:changed' event
+   * after anything was added to the path folder
+   * @param {any} path
+   * @param {any} delay – TBD
+   * @memberOf DirWathce
+   */
+  watch(dirname, delay) {
+    clearInterval(this.interval);
+    const folderPath = `${path.join(__dirname, '../', dirname)}/`;
+    this.files = this.watcher(folderPath);
+    this.interval = setInterval(() => {
+      const list = this.watcher(folderPath);
+      const delta = list.filter((filename, i) => this.files[i] !== (filename));
```

Comment:
don't use indexes, use nested find or convert lists to hashmaps. your approach won't work in case if file would be removed or new file appear at the middle of list.

Reply:
you can fix this in scope of 3rd task